docusaurus/docs/e2e-test-a-plugin/selecting-ui-elements.md

```diff
@@ -12,6 +12,10 @@ keywords:
 sidebar_position: 20
 ---
 
+# Select UI elements
+
+This guide explains the role of selectors in Grafana end-to-end testing and how to use them with multiple versions of Grafana.
```

Comment:
Good idea to have an intro. 
```suggestion
This guide explains the role of selectors in Grafana end-to-end testing and how to use them to safely interact with UI elements across multiple versions of Grafana.
```

Something like that maybe?

Reply:
done